app/assets/javascripts/modules/dir_toggle.js

```diff
@@ -0,0 +1,48 @@
+// Module adds behavior to Preview/Close preview for file objects
+
+(function( global ) {
+  var Module = (function() {
+
+    return {
+      init: function() {
+        document.querySelectorAll('.sul-embed-container').forEach(function(container){
+          DirToggle.addDirToggleBehavior(container);
+        });
+      },
+      addDirToggleBehavior: function(container) {
+        container.querySelectorAll('[data-sul-embed-dir-toggle="true"]').forEach(function(toggle){
+          toggle.addEventListener('click', function(){
+            DirToggle.toggleDirToggleIcon(toggle);
+            DirToggle.toggleDirWindow(toggle);
+          });
+        });
+      },
+      toggleDirToggleIcon: function(toggle) {
+        var icon = toggle.querySelector('i');
+        if(icon.classList.contains('sul-i-arrow-right-8')) {
```

Comment:
Can you add a class here like `open` when it is open and change the icon in the css?

Reply:
Same here too: https://github.com/sul-dlss/sul-embed/blob/master/app/assets/javascripts/modules/file_preview.js#L24-L29

Reply:
Please don't use existing code as a good example of what we want in the future. Things were done differently 8 years ago because they had requirements that no longer exist.